cns/api.go

```diff
@@ -25,6 +29,27 @@ const (
 	V2Prefix                      = "/v0.2"
 )
 
+// HTTPService describes the min API interface that every service should have.
+type HTTPService interface {
+	common.ServiceAPI
+	SendNCSnapShotPeriodically(int, chan bool)
+	SetNodeOrchestrator(*SetOrchestratorTypeRequest)
+	SyncNodeStatus(string, string, string, json.RawMessage) (int, string)
+	GetAvailableIPConfigs() []IPConfigurationStatus
+	GetPodIPConfigState() map[string]IPConfigurationStatus
+	MarkIPsAsPendingTransacted(numberToMark int) (map[string]SecondaryIPConfig, error)
```

Comment:
Can we remove Transacted? All the APIs exposed from this interface are expected to be transacted.

Reply:
sure